libs/api/database/src/lib/content/stores/content.store.ts

```diff
@@ -1,73 +1,87 @@
-import { isNullOrUndefined } from '../../util';
-import { NotificationsService } from '../notifications/notifications.service';
-import { SectionsStore } from '../sections/sections.store';
-import { ContentDocument } from './content.schema';
-import { NotificationKind } from '@dragonfish/shared/models/notifications/notification-kind';
-import { UnsubscribeResult } from '../notifications/unsubscribe-result.model';
-import { SectionsDocument } from '../sections/sections.schema';
-import { UsersStore } from '../users/users.store';
-import { ApprovalQueueStore } from '../approval-queue/approval-queue.store';
+import { JwtPayload } from '@dragonfish/shared/models/auth';
+import { BlogsContentDocument, NewsContentDocument } from '../schemas';
+import { NotificationKind } from '@dragonfish/shared/models/notifications';
+import { BlogsStore } from './blogs.store';
+import { NewsStore } from './news.store';
+import { ProseStore } from './prose.store';
+import { PoetryStore } from './poetry.store';
+import { isNullOrUndefined } from '@dragonfish/shared/functions';
+import { UsersStore } from '@dragonfish/api/database/users';
+import { NotificationsService, UnsubscribeResult } from '@dragonfish/api/database/notifications';
+import { ApprovalQueueStore } from '@dragonfish/api/database/approval-queue';
 import { MigrationForm } from '@dragonfish/shared/models/migration';
-import { ReadingHistoryStore } from '../reading-history/reading-history.store';
-import { NewsStore } from './news/news.store';
-import { PoetryStore } from './poetry/poetry.store';
-import { ProseStore } from './prose/prose.store';
-import { BlogsStore } from './blogs/blogs.store';
-import { BlogsContentDocument } from './blogs/blogs-content.document';
-import { NewsContentDocument } from './news/news-content.document';
-
+import { PublishSection, SectionForm } from '@dragonfish/shared/models/sections';
+import { SectionsStore } from './sections.store';
+
+/**
+ * ## Content Store
+ *
+ * Functions that aggregate together multiple actions for creating and updating content.
+ */
 @Injectable()
 export class ContentStore {
     constructor(
-        @InjectModel('Content') private readonly contentModel: PaginateModel<ContentDocument>,
-        private readonly blogsService: BlogsStore,
-        private readonly newsService: NewsStore,
-        private readonly poetryService: PoetryStore,
-        private readonly proseService: ProseStore,
+        @InjectModel('Content') private readonly content: PaginateModel<ContentDocument>,
+        @InjectModel('Sections') private readonly sections: Model<SectionsDocument>,
+        private readonly users: UsersStore,
+        private readonly blogContent: BlogsStore,
+        private readonly newsContent: NewsStore,
+        private readonly proseContent: ProseStore,
+        private readonly poetryContent: PoetryStore,
         private readonly sectionsService: SectionsStore,
```

Comment:
super small nit: this variable should probably be called `sections` or `sectionsStore` to fit the new theme